src/Api/Controllers/NotificationController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Api.Utility;
+using BLL.Managers;
+using BOL.Data;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Api.Controllers
+{
+    public class NotificationsController : ApiController
```

Comment:
We need an additional POST Method to set "IsReadOrNot" that allows the client to set the value